### PR TITLE
levedb / Update bem-components

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @gela-d

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "bem-core": "git://github.com/bem/bem-core.git#ff5efbef8adc1f38b730c2d0e5f405e0b0d3e3d9",
-    "bem-components": "git://github.com/bem/bem-components.git#b2e0a63522f0636da40996eb6d4042c09f48e1ff"
+    "bem-core": "2.5.0",
```
Comment (tadatuta) [inline bower.json:7]: unnecessary line

Comment (tadatuta) [inline bower.json:7]: everything else looks fine

**Comment by gela-d** [comment]
:+1:

File: bower.json (hunk 1 above)
Comment (tavriaforever) [inline bower.json:7]: open source file and I could not found unnecessary line

**Pull request merged**